flixel/util/FlxPath.hx

```diff
@@ -698,4 +778,17 @@ class FlxPath implements IFlxDestroyable
 		}
 	}
 	#end
+	
+	private function get_nodes():Array<FlxPoint>
+	{
+		if (_nodes == null)
+			_nodes = new Array<FlxPoint>();
```

Comment:
It seems the only reason this is a property is to avoid `_nodes` being null. In that case, we could just initialize `_nodes` to `[]`. It's not like `start()` allows `null` to be assigned anyway.